frontend/src/toolbar/button/Circle.tsx

```diff
@@ -107,7 +107,7 @@ export function Circle({
                           {
                               zIndex,
                               position: 'absolute',
-                              top: 0,
+                              top: 15,
```

Comment:
Hm, this might have some unintended consequences. Best to consider this `<Circle />` a library that we use and only control it from the outside. 

The right place to update the coordinates is here:
https://github.com/PostHog/posthog/blob/f9a35e2dbfcef60953d012954c28e180aaa65eae/frontend/src/toolbar/button/ToolbarButton.tsx#L176

However, it might just be fine to leave them as they are since we planned to do some bigger changes to the toolbar soon.

Reply:
Got it - will revert to original 0 here.